core/src/scss/utilities/variables/core/_modular-spacing.scss

```diff
@@ -46,3 +46,27 @@ $su-modular-spacing-multiplier: (
   8: 7,
   9: 9
 ) !default;
+
+///
+/// The modular spacing values from step -4 to 10 at 3 device width ranges.
+///
+/// @name su-modular-spacing-map
+///
+/// @group variable
+$su-modular-spacing-map: (
+  -4: (mobile: 0.3rem, tablet: 0.3rem, desktop: 0.3rem), // Consider removing
+  -3: (mobile: 0.5rem, tablet: 0.6rem, desktop: 0.6rem), // Consider removing
+  -2: (mobile: 0.8rem, tablet: 0.9rem, desktop: 1rem), // Consider removing
+  -1: (mobile: 1.1rem, tablet: 1.2rem, desktop: 1.3rem), // Consider removing
```

Comment:
These could be useful on large things.

Reply:
If it's `em`, then I'd agree, but it's `rem` so it's only 1 pixel difference (or no difference) in most of these above 👀